src/developer_config.yaml

```diff
@@ -37,6 +38,7 @@ network_paths:
   outliers_path: "R:/BERD Results System Development 2023/DAP_emulation/outliers"
   manual_outliers_path: "R:/BERD Results System Development 2023/DAP_emulation/outliers/manual_outliers/manual_outlier_2023-08-22_v45.csv"
```

Comment:
Can you change this file to "manual_outlier_2023-08-29_v67.csv" which is a much more recent one with a more accurate amount of outliers?